### DISCUSSION TITLE
How do you add a new theme to a site?

### DISCUSSION THREAD

**Discussion by aboweFIS** [OrchardCMS/OrchardCore#16727] [Q&A] [answered]
For example, if I want to add the [Orchard Base Theme](https://github.com/Lombiq/Orchard-Base-Theme) and start working with it. There's directions for making new sites and making new themes, but I can't find anything about how to simply install one to an existing site?

**Comment by agriffard** [comment]
Add the theme to your solution.
Reference the .csproj of the theme in your project using OC.
Once relaunched, go to Admin and then in menu Design > Themes (/Admin/Themes).
The theme will be displayed and you will be able to click on 'Make  Current'.

**Reply by aboweFIS** [reply]
Thank you, adding the theme to my solution is the step I don't know how to do.

**Reply by agriffard** [reply, marked as answer]
- Copy the source code of the theme in the root folder in Windows explorer. 
- Add the theme project in the solution (Right click on solution in Visual Studio, Add existing project).
- Make a reference of the theme in the OC project (https://learn.microsoft.com/en-us/visualstudio/ide/how-to-add-or-remove-references-by-using-the-reference-manager?view=vs-2022). 
- You can also drag and drop the theme in Solution explorer (https://www.meziantou.net/visual-studio-tips-and-tricks-add-project-reference-using-drag-drop.htm).

**Reply by aboweFIS** [reply]
THANK YOU!! :)
